### PR TITLE
feat(vso): Update examples

### PR DESCRIPTION
**Pull request by goock**
The PR upgrade examples for Virtual Server.
All examples were tested.

@microcosem I don't think we need to update anything in the doc but just let you know I'm updating the examples.

### CODE REVIEW

File: virtual-server/examples/curl/run.sh [hunk 1]
```diff
@@ -183,9 +183,6 @@ VS_NAME=vs-example
 echo -e "\n------------- CREATE VS -----------------------------------"
 create_vs ${VS_NAME}
 
-echo -e "\n------------- START VS -----------------------------------"
```
Comment (goock) [inline virtual-server/examples/curl/run.sh:186]: I simplified the script and removed the start, we have one later.

File: virtual-server/examples/python/kubevirtclient.py [hunk 2]
```diff
@@ -242,7 +242,12 @@ def start(self, namespace, name, **kwargs):
         """
         kwargs['_return_http_data_only'] = True
         version = self.version(namespace, name)
-        return self.kubevirt_api(namespace, name, 'start', version=version, **kwargs)
+        while True:
```
Comment (goock) [inline virtual-server/examples/python/kubevirtclient.py:245]: After the stop command, sometimes VMI is still running and returns 409.

File: virtual-server/examples/python/main.py [hunk 3]
```diff
@@ -104,22 +104,19 @@
 print(vsclient.create(my_virtualserver))
 print(f'VirtualServer status: {vsclient.ready(namespace, name)}')
 
+# Stop the Virtual Machine Instance to apply changes.
```
Comment (goock) [inline virtual-server/examples/python/main.py:107]: Same, I simplified the script, we have another stop later.

File: virtual-server/examples/python/kubevirtclient.py (hunk 2 above)
Comment (YitzyD) [reply virtual-server/examples/python/kubevirtclient.py:245]: Only really if the VM is quickly started/stopped. Good check nonetheless

**Review by YitzyD — APPROVED** [review]

**Pull request merged**